src/routes/files.ts

```diff
@@ -37,12 +37,14 @@ export default async function files(fastify: FastifyInstance) {
       const result = results.get(place.type) || [];
       result.push(record);
       results.set(place.type, result);
+      if (!hierarchyProps.has(place.type)) {
+        hierarchyProps.set(place.type, Object.keys(place.hierarchyProperties));
+      }
     });
     const zip = new JSZip();
     results.forEach((places, contactType) => {
-      const parent = Config.getParentProperty(contactType);
       const columns = [
-        parent.friendly_name,
+        ...hierarchyProps.get(contactType)!!,
```

Comment:
the columns in the resulting spreadsheet are the `property_name`. Should they be the `friendly_name`?

Reply:
done